cmd/utils/flags.go

```diff
@@ -903,6 +903,21 @@ var (
 		Usage: "When enabled a RPC node can use the L2 to build the InfoTree.",
 		Value: false,
 	}
+	Hardfork = cli.StringFlag{
+		Name:  "zkevm.hardfork",
+		Usage: "Hermez | Ethereum. Default hermez. Ethereum enabled the traditional hardfork selection system.",
```

Comment:
let's lowercase?

Reply:
probably make the available options clearer in this case e.g. - Usage: "Values { hermez 
| ethereum }. Default, hermes. ...."